caffe2/contrib/prof/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@ if (USE_PROF)
     )
   endif()
 
+  file(GLOB_RECURSE Caffe2_CONTRIB_PROF_CPU_CPP_TEST_SRCS *test.cc)
+
   set(Caffe2_CPU_SRCS ${Caffe2_CPU_SRCS} ${Caffe2_CONTRIB_PROF_CPU_SRCS} PARENT_SCOPE)
+  set(Caffe2_CPU_TEST_SRCS ${Caffe2_CPU_TEST_SRCS} 
+      ${Caffe2_CONTRIB_PROF_CPU_CPP_TEST_SRCS} PARENT_SCOPE)
```

Comment:
is this a new set of test files? you'll need to propagate that up into the parent cmakelists.txt

you'll also want to add make sure they end up copied into the part that builds the binaries https://github.com/pytorch/pytorch/blob/master/caffe2/CMakeLists.txt#L246

Reply:
whoops, nevermind this is totally fine!